server.js

```diff
@@ -27,6 +27,34 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+// Returning all the books (objects) from the array 
+app.get('/books', (req, res) => {
+  const { page } = req.query
+  const startIndex = 20 * +page
+  res.json(books.slice(startIndex, startIndex + 20))
+})
+
+//request.params is an object containing properties to the named route
+// Return an individual book, using :id as a placeholder for the bookID number
+app.get('/books/:id', (req, res) => {
+  const id = req.params.id
+  const bookId = books.filter((item) => item.bookID === +id) // + means turning string into a number
+  res.json(bookId)
```

Comment:
I think this line can be removed since the response is sent in the condition below